src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/GetExecutionPlanRequest.cs

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.ShowPlan
+{
+    public class GetExecutionPlanParams
+    {
+        public ExecutionPlanGraphFile GraphFile { get; set; }
```

Comment:
nit: not sure "File" is the right name here, i.e., does this content have to come from a file or could it come over the network, etc.  I think we've generally used the term "Info" to describe blobs of data, such as `ExecutionPlanGraphInfo`.